src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
-);
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => {
+  const {
+    name = 'comment',
```

Comment:
The default value for 'name' is set to 'comment', which might not be meaningful as a name. Consider setting the default value to something like 'Anonymous' or not providing a default name at all.